app/src/app/routes/Router.tsx

```diff
@@ -27,6 +28,9 @@ const Router = (): JSX.Element => {
           <Route exact path="/source/:sourceId">
             <SourceMappings />
           </Route>
+          <Route exact path="/source/:sourceId/createMapping">
```

Comment:
could this route be `/source/:sourceId/mapping` and when the mapping is created, it would jump to `/source/:sourceId/mapping/:mappingId` ?

Reply:
agreed, I think that's better!